kotlin-analysis/intellij-dependency/build.gradle.kts

```diff
@@ -35,19 +35,17 @@ fun jpsModel() = zipTree(jpsStandalone.singleFile).matching {
 }
 
 dependencies {
-    val kotlin_plugin_version: String by project
-    api("org.jetbrains.kotlin:common:$kotlin_plugin_version")
-    api("org.jetbrains.kotlin:idea:$kotlin_plugin_version") {
+    api(libs.kotlinPlugin.common)
+    api(libs.kotlinPlugin.idea) {
         isTransitive = false
     }
-    api("org.jetbrains.kotlin:core:$kotlin_plugin_version")
-    api("org.jetbrains.kotlin:native:$kotlin_plugin_version")
+    api(libs.kotlinPlugin.core)
+    api(libs.kotlinPlugin.native)
 
-    val idea_version: String by project
-    intellijCore("com.jetbrains.intellij.idea:intellij-core:$idea_version")
+    intellijCore(libs.jetbrainsIntelliJ.core)
     implementation(intellijCoreAnalysis())
 
-    jpsStandalone("com.jetbrains.intellij.idea:jps-standalone:$idea_version")
+    jpsStandalone(libs.jetbrainsIntelliJ.jpsStandalone)
```

Comment:
Qodana is complaining about unstable API usage

Reply:
I'll suppress it for now, but tbh I think this bit of code probably needs a refactor - the JARs can be fetched via existing configurations (possibly). 

I think these dependencies are related to https://github.com/Kotlin/dokka/issues/2889 too? So might they be removed in the future?

Reply:
Yep, suppressing it is totally fine. These should be removed/refactored quite soon, it's a blocked for 1.8.20